tests/integration-tests/conftest.py

```diff
@@ -112,8 +114,10 @@ def pytest_addoption(parser):
     parser.addoption(
         "--no-delete", action="store_true", default=False, help="Don't delete stacks after tests are complete."
     )
-    parser.addoption("--benchmarks-target-capacity", help="set the target capacity for benchmarks tests", type=int)
-    parser.addoption("--benchmarks-max-time", help="set the max waiting time in minutes for benchmarks tests", type=int)
+    parser.addoption("--benchmarks", action="store_true", default=False, help="enable benchmark tests")
+    # parser.addoption("--benchmarks-target-capacity", help="set the target capacity for benchmarks tests", type=int)
+    # parser.addoption("--benchmarks-max-time",
```

Comment:
Is there a reason these are staying here and commented? If so let's add a comment to how they are useful, otherwise let's remove them.

Reply:
I'd remove these. Because demartinofra, the original author, agreed to change the scaling test approach